slsReceiverSoftware/src/BinaryFile.cpp

```diff
@@ -191,6 +191,13 @@ void BinaryFile::CreateMasterFile(bool masterFileWriteEnable,
                 "Dbit Offset                : %d\n"
                 "Dbit Bitset                : %lld\n"
                 "Roi (xmin, xmax)           : %d %d\n"
+                "Exptime1 (ns)              : %lld\n"
```

Comment:
I guess we want to keep the master file the same for all detectors? But now we write many fields that does not exists for most detectors

Reply:
Yes, this is something I would like to change in the future.